test/functional/lua/snippet_spec.lua

```diff
@@ -230,14 +231,20 @@ describe('vim.snippet', function()
   end)
 
   it('updates snippet state when built-in completion menu is visible', function()
-    test_expand_success({ '$1 = function($2)\n$3\nend' }, { ' = function()', '', 'end' })
+    local curbuf = api.nvim_get_current_buf()
+    api.nvim_set_option_value('sw', 2, { buf = curbuf })
```

Comment:
This should be a separate test, with its own description.

Reply:
I just feel this case is enough. But there is no harm in adding a newline of tabs and spaces.

Reply:
The point is about being explicit _what_ each test is testing. Testing multiple things at the same time means failures will be harder to interpret.

Reply:
emm seems like my test case is missing, stash restore my case lol i will readd it.